src/btree.c

```diff
@@ -2814,6 +2818,11 @@ btree_pack_loop(btree_pack_req *req, // IN/OUT
 {
    log_trace_key(key, "btree_pack_loop");
 
+   if (message_is_invalid_user_type(msg)) {
```

Comment:
Why not return the error as the return value of `btree_pack_loop`?

Reply:
@rtjohnso -- I felt like that an error handling like this could be possibly used in other functions related to btree packing. But now I think that returning the error is a better way for this because we don't need to change the btree interface.

Reply:
@ajhconway -- I think that it is better than adding a new error type. Let me do that.